abi.md

```diff
@@ -351,13 +351,13 @@ circuit Bar:
 When compiled to Verilog, this creates two Verilog modules.
 Verilog module `Bar` will be sensitive to the following defines:
 
-    layer_Bar$Layer1$Layer2
-    layer_Bar$Layer1$Layer2$Layer3
+    layer$Layer1$Layer2
+    layer$Layer1$Layer2$Layer3
 
 Verilog module `Baz` will be sensitive to the following defines:
 
-    layer_Baz$Layer1$Layer2
-    layer_Baz$Layer1$Layer2$Layer3
+    layer$Layer1$Layer2
+    layer$Layer1$Layer2$Layer3
```

Comment:
Can this text be combined to indicate that both are sensitive to the same defines? This mattered when the defines were different, but not now.